docs/src/components/home/Hero.tsx

```diff
@@ -69,7 +69,11 @@ const grid = [
     height: 119,
   },
   {
-    ui: <ThemeTabs />,
+    ui: (
+      <Box sx={{ transform: { xs: 'scale(1.11)', lg: 'scale(1)' } }}>
+        <ThemeTabs />
+      </Box>
```

Comment:
We could almost consider this a bug in the Tabs; the solution

<img width="270" alt="Capture d’écran 2021-08-04 à 18 17 58" src="https://user-images.githubusercontent.com/3165635/128217237-21f55730-14c4-4a09-80ee-c1fa19ccc323.png">

https://github.com/mui-org/material-ui/blob/873e79c790c2fbd3f5985ed3811cffdd4e7adfc1/packages/material-ui/src/Tabs/Tabs.js#L405

Reply:
👍